x-pack/plugins/security_solution/public/common/components/events_viewer/use_timelines_events.tsx

```diff
@@ -270,11 +267,6 @@ export const useTimelineEventsHandler = ({
                   setFilterStatus(request.filterStatus);
                   setLoading(false);
 
-                  searchSubscription$.current.unsubscribe();
-                } else if (isErrorResponse(response)) {
```

Comment:
@nreese , since `isErrorResponse` is already handled by `search` api and it is throwing, this code should be moved to `error` of subscription. no? I have added suggestion for that in next comment on the `error` section.